test cases/d/13 declare dep/meson.build

```diff
@@ -3,7 +3,7 @@ project('meson-d-sample', 'd',
 )
 
 my_dep = declare_dependency(
-    d_module_versions: ['TestVersion', 1],
+    d_module_versions: ['TestVersion'],
     d_import_dirs: include_directories('views'),
 )
```

Comment:
Shouldn't this also be an if/else?

Reply:
I chose to do it simpler because:

1) Nothing broke when I took out the number
2) Numbers as values are deprecated in any case